boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -29,6 +29,10 @@ config BOARD_ENABLE_DCDC_HV
 	select SOC_DCDC_NRF53X_HV
 	default y
 
+config BOARD_ENABLE_USB_CDC
```

Comment:
At the end of commit message:
... as the UART on this **board** is not connected.

Reply:
Fixed